File: features/lender/src/main/java/com/nivasafinance/features/lender/lenderoffice/dto/LenderOfficeReponseData.java
package com.nivasafinance.features.lender.lenderoffice.dto;

import com.nivasafinance.common.dto.AddressData;
import com.nivasafinance.features.lender.lenderoffice.enums.LenderOfficeStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LenderOfficeReponseData {
    private UUID id;
    private String name;
    private String key;
    private String lenderKey;
    private AddressData address;
    private LenderOfficeStatus status;
}

File: features/lender/src/main/java/com/nivasafinance/features/lender/lenderoffice/dto/LenderOfficeReponseData.kt
package com.nivasafinance.features.lender.lenderoffice.dto

import com.nivasafinance.common.dto.AddressData
import java.util.*

data class LenderOfficeReponseData(
    val id: UUID,
    val name: String,
    val key: String,
    val lenderKey: String,
    val address: AddressData?
)
